### DISCUSSION TITLE
Add link for database Checkpoint to GUI

### DISCUSSION THREAD

**Discussion by Voodoo-Chia** [Chia-Network/chia-blockchain-gui#2379] [Ideas]
Follow up on Discord discussion about adding database checkpoint to the chia GUI, especially for new users.

Because actually integrating it into the GUI is much more complicated, the proposal is to just add a link with description to the Chia GUI (somewhere clearly visible near the syncing counter on the full node tab) to make sure users are aware of the option to download a database checkpoint instead of syncing the whole chain from scratch. 

![checkpoint chia](https://github.com/Chia-Network/chia-blockchain/assets/88035618/f9d23d98-9e77-46ed-bb48-de6f1ad078c9)

**Comment by BrandtH22** [comment]
@Voodoo-Chia , note I transferred this enhancement request to the GUI repo for better visibility 👍
